lib/hacluster.pm

```diff
@@ -210,7 +210,6 @@ sub ha_export_logs {
     my @y2logs;
 
     # Extract HA logs and upload them
-    select_console 'root-console';
```

Comment:
Why?

Reply:
Seems to be not useful, but yes I agree than I can keep it "in case of".
I will re-run a test with it, to be sure.

Reply:
`select_console` was part of the weird failures I had... It works without it on my last test, so I prefer to remove it for now.